api/serializers/student/dashboard.py

```diff
@@ -65,3 +68,27 @@ class Meta(CommonDashboardStudentSerializer.Meta):
             "personal_info",
             "groups",
         )
+
+
+class DashboardStudentTransferSerializer(serializers.Serializer[Any]):
+    to_group_id = serializers.IntegerField()
+    from_group_id = serializers.IntegerField()
+
+    def validate(self, attrs: dict[str, Any]) -> dict[str, Any]:
+        try:
+            to_group = Group.objects.get(pk=int(attrs["to_group_id"]))
```

Comment:
Reason?

Reply:
Because the meaning of try statement is to cover the spot where exception could occur. If we don't wait an exception in some line(s) of code, why we need to put it into try..except block?

Reply:
It will also make a code more readable, you will remove unnecessary nesting, and it will be easier to understand where exception could be raised

Reply:
First point is reasonable, less error prone but in my sense code is more readable when you use try-catch as an abstraction but, yes, in this case you have another exception inside which is somehow confusing